presto-parser/src/test/java/com/facebook/presto/sql/parser/TestSqlParser.java

```diff
@@ -1636,6 +1640,425 @@ public void testDropColumn()
         assertStatement("ALTER TABLE IF EXISTS foo.t DROP COLUMN IF EXISTS c", new DropColumn(QualifiedName.of("foo", "t"), identifier("c"), true, true));
     }
 
+    @Test
+    public void testCreateTableWithConstraints()
```

Comment:
This look like a good candidate to turn into a parameterized](https://testng.org/doc/documentation-main.html#parameters-dataproviders) test

Reply:
I'll take another crack at this. I remember trying to do this earlier - but the behavior was different between the constraints in many cases

Reply:
Most of these tests involve a combination of primary keys and unique constraints. They do not lend themselves to the parametrized framework. IMO it is not worth it. We can revisit this if we expand this test to include many more cases